module_auto_update/wizards/module_upgrade.py

```diff
@@ -30,4 +30,12 @@ def upgrade_module(self):
         # Compute updates by checksum when called in @api.model fashion
         if not self:
             self.get_module_list()
-        return super(ModuleUpgrade, self).upgrade_module()
+        # Get base adddon status before updating
+        base = self.env["ir.module.module"].search([("name", "=", "base")])
+        pre_state = base.state
+        result = super(ModuleUpgrade, self).upgrade_module()
+        # Update base addon checksum if its state changed
+        base.invalidate_cache()
+        if base.state != pre_state:
+            base.latest_version = base.latest_version
```

Comment:
Or maybe a comment indicating why weird assignment is necessary if this is some edge case?